package-scripts.js

```
@@ -23,13 +23,13 @@ module.exports = {
       // supports modules in the VM API or the import is removed from prettier.
       default: crossEnv(
-        'NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules jest --coverage'
+        'NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules jest --coverage',
       ),
       update: crossEnv(
-        'NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules jest --coverage --updateSnapshot'
+        'NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules jest --coverage --updateSnapshot',
       ),
       watch: crossEnv(
-        'NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules jest --watch'
+        'NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules jest --watch',
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)